scripts/embed-dashboard-ui.ps1

```diff
@@ -0,0 +1,45 @@
+$DIR = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$BASE_DIR = (get-item $DIR).parent.FullName
+$CACHE_DIR = Join-Path($BASE_DIR) "\dashboard_asset_cache"
+
+echo $DIR
+
+echo '+ Create cache directory'
+
+mkdir -p $CACHE_DIR
+
+echo '+ Fetch Dashboard Go module'
+go mod download
+
+echo '+ Discover Dashboard UI version'
+
+$DASHBOARD_DIR=$(go list -f "{{.Dir}}" -m github.com/pingcap-incubator/tidb-dashboard)
+echo "  - Dashboard directory: $DASHBOARD_DIR"
+
+$DASHBOARD_UI_VERSION="20200427_1"
+echo "  - Dashboard ui version: $DASHBOARD_UI_VERSION"
+
+echo '+ Check embedded assets exists in cache'
+$CACHE_FILE= Join-Path($CACHE_DIR) \embedded-assets-golang-${DASHBOARD_UI_VERSION}.zip
+
+if (Test-Path $CACHE_FILE ){
+  echo "  - Cached archive exists: $CACHE_FILE"
+}
+else{
+  echo '  - Cached archive does not exist'
+  echo '  - Download pre-built embedded assets from GitHub release'
+  $DOWNLOAD_URL="https://github.com/pingcap-incubator/tidb-dashboard/releases/download/ui_release_${DASHBOARD_UI_VERSION}/embedded-assets-golang.zip"
+  echo "  - Download ${DOWNLOAD_URL}"
+  curl -UseBasicParsing "${DOWNLOAD_URL}" > embedded-assets-golang.zip

```

Comment:
The file was not successfully downloaded.

Reply:
fixed.